libs/sdk-react-native/src/index.tsx

```diff
@@ -90,6 +90,46 @@ export enum BuyBitcoinProvider {
     MOONPAY = "moonpay"
 }
 
+export type OpeningFeeParams = {
+    minMsat: number
+    proportional: number
+    validUntil: string
+    maxIdleTime: number
+    maxClientToSelfDelay: number
+    promise: string
+}
+
+export type OpeningFeeParamsMenu = {
+    values: OpeningFeeParams[]
+}
+
+export type ReceivePaymentRequest = {
+    amountSats: number
+    description: string
+    preimage?: Uint8Array
```

Comment:
Better for the preimage type to be `Uint8Array | number[]`, as the preimage should be converted to an array to be passed over the RN bridge.

Reply:
Done as part of 2b40b13c05d6a84425019c0c35d88d7289705e3c